src/resources/boot.lua

```diff
@@ -124,7 +124,8 @@ function lovr.boot()
       drivers = { 'openxr', 'oculus', 'vrapi', 'pico', 'openvr', 'webxr', 'desktop' },
       supersample = false,
       offset = 1.7,
-      msaa = 4
+      msaa = 4,
+      overlay = false,
```

Comment:
My only style comment is to omit the trailing commas here and in headset_openxr.c since they aren't currently used in the codebase

Reply:
Oki, they do have a advantage as when you add a element you only get a single line added, compared to a line removed and two lines added. :)